source/events/events.rst

```diff
@@ -0,0 +1,295 @@
+.. include:: ../references.txt
+
+.. iact_events:
+
+``EVENTS`` extension
+====================
+
+The ``EVENTS`` extension is a binary FITS table that contains an event list.
+Each row of the table provides information that characterises one event.
+The mandatory and optional columns of the table are listed below. In
+addition, a list of header keywords providing metadata is specified.
+Also here there are mandatory and optional keywords.
+The suggested extension name of the binary table is ``EVENTS``.
+
+
+Mandatory columns
+-----------------
+
+* ``EVENT_ID`` tform: ``1J``
+    * Event identification number at the DL3 level
+      (lower data levels could be different, see note below).
+* ``TIME`` tform: ``1D``, unit: s
+    * Time stamp of event in instrument specific MJD time reference. See the
+      header keywords ``MJDREFI`` and ``MJDREFF`` for the zero point of
+      the reference time.
+      See also the `OGIP event list`_ standard.
+* ``RA`` tform: ``1E``, unit: deg
+    * Reconstructed event Right Ascension (see :ref:`sky-coordinates-radec`).
+      See also `HFWG Recommendation R3`_ for the OGIP standard.
+* ``DEC`` tform: ``1E``, unit: deg
+    * Reconstructed event Declination (see :ref:`sky-coordinates-radec`).
+      See also `HFWG Recommendation R3`_ for the OGIP standard.
+* ``ENERGY`` tform: ``1E``, unit: TeV
+    * Reconstructed event energy.
+      See also the `OGIP event list`_ standard.
+* ``DETX`` tform: ``1E``, unit: deg
+    * Reconstructed event X-coordinate in detector system
+      (nominal system, see :ref:`sky-coordinates-fov`).
+      See also the `OGIP event list`_ standard.
+* ``DETY`` tform: ``1E``, unit: deg
+    * Reconstructed event Y-coordinate in detector system
+      (nominal system, see :ref:`sky-coordinates-fov`).
+      See also the `OGIP event list`_ standard.
+* ``EVENT_TYPE`` tform: ``32X``
+    * Event quality partition.
+
+      .. note::
+         It needs to be discussed whether this should be a mandatory column
+         or an optional column. The partition of events into quality classes
+         seems sufficiently fundamental so that all IACT should in principle
+         support this. For example event multiplicity could be translated
+         into quality partitions, and the IRFs computed accordingly.
+
+
+Optional columns
+----------------
+
+.. note::
+   None of the following columns is required to be part of an ``EVENTS``
+   extension. Any software using these columns should first check whether the
+   columns exist, and warn in case of their absence.
```

Comment:
Well, that's not what the text says. It applies to "software **using** these columns", so any software ignoring these columns can stay silent. But I recognised that it does not say what a software that is ignoring these columns should do in case that these columns exist. So I propose:

>    None of the following columns is required to be part of an ``EVENTS``
>    extension. Any software **using** these columns should first check whether the
>    columns exist, and warn in case of their absence. Any software **ignoring**
>    these columns should make sure that their presence does not detoriate the
>    functioning of the software.

Agree that this can be moved upfront, but not sure where you are with the restructuring of the documentation, so I propose to merge this in first and then create an issue to move this note upfront.